learn/features/governance.md

```diff
@@ -26,6 +26,34 @@ Moonbeam作为去中心化网络，将由核心开发者、应用程序开发者
  - **锁定期** —— 赢得投票用户的代币锁定期（提案执行后）。在此期间，用户仍可使用锁定代币进行质押或投票
  - **委托** —— 将自己的投票权委托给其他账户，以积累一定信念值的行为
 
+=== "Moonbeam"
+    |         变量         |  |                                                            Value                                                             |
```

Comment:
should value be translated?

Reply:
same goes for the Moonriver and Moonbase Alpha

Reply:
I would maybe do a search in the repo for all use cases of "Variable" and Value" and update those cause I actually just noticed that "Variable" isn't translated here https://github.com/PureStake/moonbeam-docs-cn/pull/44/files#diff-c9c61564fa95109849566c20d6079bb70faa093f2ccb926fad1ad3a43ef123c9R39-R45